services/api/database/migrations/20240114000000_environment_services.js

```diff
@@ -28,5 +31,6 @@ exports.down = async function(knex) {
         table.dropColumn('updated');
         table.dropColumn('created');
     })
+    .raw(`ALTER TABLE environment_service DROP INDEX service_environment;`)
```

Comment:
I'd convert this back to the native knex statement, and put in inside the single down, just to avoid reliance on raw sql where possible?